common/src/sql/dbinit.sql

```diff
@@ -1419,6 +1419,20 @@ CREATE INDEX ON omicron.public.update_available_artifact (
     targets_role_version
 );
 
+/*
+ * System updates
+ */
+CREATE TABLE omicron.public.system_update (
+    /* Unique identifier for this update package */
+    id UUID PRIMARY KEY,
+    time_created TIMESTAMPTZ NOT NULL,
+    time_modified TIMESTAMPTZ NOT NULL
+
+    /* Unique semver version */
+    /* TODO: If the version is really supposed to be unique, we could make it the PK? */
+    version STRING(40) NOT NULL,
```

Comment:
We could do ser/de on strings but it's kind of silly. Would be easier to store major, minor, patch as numbers and run them through `SemverVersion::new(major, minor, patch)` on DB read.